lib/unique_head.rb

```diff
@@ -52,6 +59,10 @@ def header(text, header_level)
       idxTrim = text.index("NEW")
       content = text[0,idxTrim]
       return "<h#{header_level} id='#{friendly_text}' type='new'>#{content} <span class='tags new'>NEW</span> </h#{header_level}>"
+    elsif(text.include? "COMINGSOON")
```

Comment:
kalau pake yang lain gimana pi namanya, kira2 nanti bakal ada orang yang taro COMINGSOON ngga ya di api docs, apa harusnya udah aman ya?

Reply:
Bebas sih keynya apa, atau solusinya bakal di pakai readme, untuk prevent pakai tulisan itu, wdyt ?

Reply:
ohh di taro glossary of keywords gitu ya
oke sihh